test/utils/computeLedgerHash.ts

```diff
@@ -38,77 +49,73 @@ describe("computeLedgerHash", function () {
   });
 
   it("given ledger without raw transactions - should throw", function () {
-    const ledger = JSON.parse(JSON.stringify(responses.getLedger.full));
-    delete ledger.transactions[0].rawTransaction;
+    const ledger = getNewLedger();
```

Comment:
Are there some merge conflict issues here? I thought I made these changes in #1575